react/components/SKUSelector/components/SelectorItem.tsx

```diff
@@ -40,22 +42,30 @@ const SelectorItem: FC<Props> = ({
   imageUrl,
   imageLabel,
   isImpossible,
+  heightImg,
+  widthImg,
 }) => {
   const discount = getDiscount(maxPrice, price)
+
+  const rootClasses = classNames(
+    styles.skuSelectorItem,
+    `${styles.skuSelectorItem}--${slug(variationValue)}`,
+    'relative di pointer flex items-center outline-0',
+    {
+      [styles.skuSelectorItemImage]: isImage,
+      'o-20': isImpossible,
+    }
+  )
+
+  const rootStyles = isImage && heightImg && widthImg ? { height: heightImg, width: widthImg, padding: 0 } : {}
```

Comment:
you might want to allow the user setting only "width", for instance, and letting the "height" follow. or, if setting only the "width" does nothing, perhaps warning on the console that it's ineffective